hackclub.com.yaml

```diff
@@ -3216,3 +3216,15 @@ portal:
   - ttl: 600
     type: A
     value: 173.255.236.153
+svis:
+  - ttl: 600
+    type: MX
+    value: mx1.improvmx.com.
+    priority: 10
+  - ttl: 600
+    type: MX
+    value: mx2.improvmx.com.
+    priority: 20
+  - ttl: 600
+    type: TXT
+    value: v=spf1 include:spf.improvmx.com ~all
```

Comment:
Policy states that subdomains of hackclub.com can't be mail-enabled unless it's for an official HQ project.

Reply:
but I saw some people using protonmail above. Are those for the official HQ Project? Anyways sorry.